core-header-panel.html

```diff
@@ -146,11 +146,11 @@
 
       <div id="mainPanel" class="layout vertical flex">
 
-        <div id="mainContainer" class-name="[[_mainContainerClass]]">
+        <div id="mainContainer" class$="[[_computeMainContainerClass(mode)]]">
           <content id="mainContent" select="*"></content>
         </div>
 
-        <div id="dropShadow" class-name="[[_dropShadowClass]]"></div>
+        <div id="dropShadow" hidden$="[[_computeDropShadowHidden(_atTop, mode, shadow)]]"></div>
```

Comment:
OTOH, having a consistent look with the shadows across elements seems like a good thing, especially since we have classes for the shadows now. But that should probably go in a different change?